source/SaltyChatServer/VoiceManager.cs

```diff
@@ -110,32 +111,38 @@ public void OnCheckVersion(IPlayer player, string branch, string version)
         [ClientEvent(Event.SaltyChat_IsTalking)]
         public void OnIsTalking(IPlayer player, bool isTalking)
         {
-            if (!VoiceManager.TryGetVoiceClient(player, out VoiceClient voiceClient))
+            if (!TryGetVoiceClient(player, out _))
                 return;
 
-            foreach (VoiceClient client in VoiceManager.VoiceClients.Values)
+            foreach (VoiceClient client in VoiceClients.Values)
             {
                 client.Player.Emit(Event.SaltyChat_IsTalking, player.Id, isTalking);
             }
         }
 
-        [ClientEvent(Event.SaltyChat_SetVoiceRange)]
-        public void OnSetVoiceRange(IPlayer player, int range)
+        [ServerEvent(Event.SaltyChat_SetVoiceRange)]
+        public void OnSetVoiceRange(IPlayer player, string range)
         {
-            float voiceRange = (float) range;
-            if (!VoiceManager.TryGetVoiceClient(player, out VoiceClient voiceClient))
+            float voiceRange;
+
+            voiceRange = 8.0f;
+
+            if (range == "Weit")
+                voiceRange = 22.0f;
+
+            else if (range == "Kurz")
+                voiceRange = 2.0f;
+
+            Console.WriteLine("VOICE RANGE TEST: " + voiceRange);

```

Comment:
Why are there no comments on any change?
No one wants to guess why you change things in a certain way and why it is better.
This change for example is error prone, due to the use of strings that aren't even english.

Debug outputs shouldn't be in a "final" script, especially such verbose ones, which will spam the server console on a production server.